msal/build.gradle

```diff
@@ -23,18 +23,18 @@ def bothTestType = "BothTest"
 
 android {
 
-    testOptions {
-        compileOptions {
-            // Flag to enable support for the new language APIs
-            coreLibraryDesugaringEnabled true
-            // Sets Java compatibility to Java 8
-            sourceCompatibility JavaVersion.VERSION_1_8
-            targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
Can we explicitly set these to 1_7 and false for the compile options?

Reply:
Same answer as the other one.

I'm inclined not to test this in the last minute (MSAL release is already overdue). Commenting out is safer.

I created a [work item](https://identitydivision.visualstudio.com/Engineering/_boards/board/t/Auth%20Client%20-%20Android/Backlog%20items/?workitem=1313685) and we can spend time to test this out thoroughly in dev.